### PR TITLE
Use funcs from shared module

### PR DESCRIPTION
**Pull request by alexhung**
Update module dependency to latest versions

Use tflog for logging

### CODE REVIEW

**Comment by alexhung** [comment]
@chb0github ☝🏼

File: pkg/project/resource_project_membership_test.go [hunk 1]
```diff
@@ -262,7 +263,7 @@ func TestAccProjectGroup(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "key", fmt.Sprintf("%s", params["project_key"])),
 					resource.TestCheckResourceAttr(resourceName, "display_name", name),
 					resource.TestCheckResourceAttr(resourceName, "description", "test description"),
-					resource.TestCheckNoResourceAttr(resourceName, "group"),
+					resource.TestCheckResourceAttr(resourceName, "group.#", "0"),
```
Comment (alexhung) [inline pkg/project/resource_project_membership_test.go:266]: The updated TF SDK doesn't like the old way any more. So have to change to this test.

**Review by chb0github — COMMENTED** [review]
Looks great

**Review by danielmkn — APPROVED** [review]
All test passed

**Review by maheshjfrog — APPROVED** [review]

**Pull request merged**